spec/game_spec.rb

```diff
@@ -0,0 +1,58 @@
+require "./lib/game"
+require "./lib/player"
+require "./lib/module"
+
+RSpec.describe Game do
+
+    let(:player1) {Player.new("player1", "X")}
+    let(:player2) {Player.new("player2", "O")}
+    let(:board) {Board.new}
+    let(:game) { Game.new(player1,player2,board) }
+    let(:piece) { "x" }
+    let(:piece2) { "o" }
+
+    describe "#check_draw" do
+
+        it "returns true and a message in case of draw and false in the opposite" do
+            
+            allow(board).to receive(:full?).and_return(true)
+
+            expect(game.check_draw).to eq(true)
```

Comment:
This test checks for a positive scenario, but you also should have a test that checks when you expect draw to equal false